variables.tf

```diff
@@ -18,10 +30,22 @@ variable "engine" {
 
 variable "engine_version" {
   type        = string
-  default     = "5.0.6"
+  default     = "6.x"
   description = "Engine version to use in the elasticache cluster"
 }
 
+variable "in_transit_encryption" {
+  type        = bool
+  default     = false
```

Comment:
Maybe default to `null`, which means you did not set the value. Seems like a good default since its an optional field.